docs/source/developer_guides/lora.md

```diff
@@ -122,6 +122,22 @@ from peft import LoraConfig
 config = LoraConfig(use_dora=True, ...)
 ```
 
+If parts of the model or the DoRA adapter are offloaded to CPU you can get a significant speedup at the cost of some temporary (ephemeral) VRAM overhead by using `ephemeral_transfers=True` in `config.runtime_config`.
+
+```py
+from peft import LoraConfig, LoraRuntimeConfig
```

Comment:
This import fails as `LoraRuntimeConfig` is not loaded to the PEFT `__init__.py`.

Reply:
D'oh, thanks. Fixed.